sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -111,6 +122,7 @@ def from_connection_string(
             account_url, file_system_name=file_system_name, directory_name=directory_name,
             credential=credential, **kwargs)
 
+    @distributed_trace
```

Comment:
Hmm I wonder if there was a reason these weren't here already? Maybe we should ask Anna about this.